dev/README.md

```diff
@@ -0,0 +1,32 @@
+# Setting up a Development Environment
+This describes how to set up a development environment suitable for working on the automation for `gatling-puppet-load-test`
+
+## What you get
+The beaker scripts in the `beaker` directory will set up a machine with a jenkins server, a copy of this repo (which contains the gatling executables), `sbt` for running gatling, and an installation of `jenkins-job-builder` configured to point at the jenkins server.
+
+## Requirements
+This guide assumes you have a centos-6 machine/VM with:
+  * ssh key authentication for the root user (for `beaker`)
+  * `puppet` installed and in the `PATH`
```

Comment:
This has bitten me like 4 times now (which, admittedly, is because I'm an idiot).  Would be really nice to have the setup scripts update the timestamp on the VM.